Update ReleasingTor.md to reflect current practice (#521)
torproject/tor: doc/HACKING/ReleasingTor.md

      "Relays", not "servers" or "nodes" or "Tor relays".

>       "Onion services", not "hidden servies".

Reviewer: ```suggestion
      "Onion services", not "hidden services".
```

   master, merge it with "-s ours" to avoid a needless version bump.

2. Make distcheck, put the tarball up in somewhere (how about your
   homedir on your homedir on people.torproject.org?) , and tell `#tor-dev`
   about it.

   If you want, wait until at least one person has built it
   successfully.  (We used to say "wait for others to test it", but our

>    CI has successfully caught this kind of errors for the last several

Reviewer: ```suggestion
   CI has successfully caught these kinds of errors for the last several
```

   years.)

3. Make sure that the new version is recommened in the latest consensus.
   (Otherwise, users will get confused when it complains to them

>    about its status.)

Reviewer 2: This part is not that straight forward since few people have access to the dirauth conf git repository. And there has been an issue some days ago about how to do that. Since 90% of the time, weasel push the changes there for which he mentioned that he would like a request from someone from the network team.

Meaning, that section should include *how* to do that and for now it simply seems "Ask weasel to add the version XXX".

Author: d336b62fd9d811c72ec8e5929804039285616102 tries to improve this

=== IV. Commit, upload, announce

1. Sign the tarball, then sign and push the git tag:

        gpg -ba <the_tarball>
        git tag -s tor-0.4.x.y-status

>         git push origin tag tor-0.4.x.y-status

Reviewer 2: Maybe put `status` like this `-<status>` because I actually got worried for a second that our tags had -status in them.

Author: d336b62fd9d811c72ec8e5929804039285616102 makes this change

   Also, email tor-packagers@lists.torproject.org.

   Mention where to download the tarball (https://dist.torproject.org).

   Include a link to the changelog.

4. Add the version number to Trac.  To do this, go to Trac, log in,
    select "Admin" near the top of the screen, then select "Versions" from
    the menu on the left.  At the right, there will be an "Add version"
    box.  By convention, we enter the version in the form "Tor:
    0.4.0.1-alpha" (or whatever the version is), and we select the date as
    the date in the ChangeLog.

5. Wait for the download page to be updated. (If you don't do this before you
   announce, people will be confused.)

6. Mail the release blurb and ChangeLog to tor-talk (development release) or
   tor-announce (stable).

   Post the changelog on the blog as well. You can generate a
   blog-formatted version of the changelog with

>       `./scripts/maint/format_changelog.py --B

Reviewer: ```suggestion
      `./scripts/maint/format_changelog.py --B`
```
